homework_7_functional_programming/functional.py

```diff
@@ -0,0 +1,28 @@
+def sequential_map(*args):
+    *functions_list, container = args
+    for function in functions_list:
+        result_list = list(map(function, container))
+    return result_list

```

Comment:
1. Превратить элемент в список можно только в самом конце, map можно передавать в другие map'ы
2. Cм. коммент к `func_chain`
```suggestion
    for function in functions_list:
        container = map(function, container)
    return list(container)

```